src/components/AddVehicle.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Container, Row, Col, Alert, Form, Button,
+} from 'react-bootstrap';
+import { MdCheck } from 'react-icons/md';
+import { AiOutlineRightCircle } from 'react-icons/ai';
+import { addVehicle } from '../redux/vehicles/vehicles';
+import '../styles/AddVehicle.scss';
+import { getAuth, uploadFile } from '../redux/uploadcare/uploadcare';
+
+const DEFAULT_VALUES = {
+  brand: 'Yamaha',
+  model: '2003',
+  description: 'Really good bike',
+  price: 10000,
+  image: 'image.jpg',
+  visible: true,
+};
+
+function AddVehicle() {
```

Comment:
- [ ] It was impossible for me to add a vehicle.  I just get greeted with a blank page and this error:

![image](https://user-images.githubusercontent.com/57421823/194246940-75cb3793-4353-49da-b1eb-af8a12ec93b3.png)

- [ ] As of now, it is impossible to reserve a car because the POST operation fails:

![image](https://user-images.githubusercontent.com/57421823/194247349-cbfe1426-9df2-4143-8169-4af1890c13aa.png)



Kindly fix these bugs to ensure your app's proper functionality. 💪